app/_components/MapContainer.tsx

```diff
@@ -27,8 +28,10 @@ function MapInner() {
   const [selectedPoiId, setSelectedPoiId] = useState<number | undefined>(
     undefined
   );
-  const [longitude] = useState<number>(139.7454); // Default camera map when user opens the app
-  const [latitude] = useState<number>(35.6586);
+
+  // Default camera map when user opens the app
+  const [longitude] = useState<number>(139.80241);
+  const [latitude] = useState<number>(35.56762);
```

Comment:
Is there a reason we're storing it in a state even when there's no setter function?